src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -1863,10 +1870,24 @@ public void execute() throws GitException, InterruptedException {
                 // Split fields into branch name, SHA1, and rest of line
                 // Fields are separated by one or more spaces
                 String[] branchVerboseOutput = line.substring(2).split(" +", 3);
+                if (branchVerboseOutput[1].equals("->")) {
```

Comment:
How is this related? I guess you are changing `getRemoteBranches` from using JGit to using CLI Git?